src/app/tests/suites/certification/Test_TC_S_2_6.yaml

```diff
@@ -52,114 +105,243 @@ tests:
+          [1701251118.633644][8779:8781] CHIP:DMG:                                 ],
+          [1701251118.633646][8779:8781] CHIP:DMG:                         },
+          [1701251118.633652][8779:8781] CHIP:DMG:
+          [1701251118.633655][8779:8781] CHIP:DMG:                 },
+          [1701251118.633661][8779:8781] CHIP:DMG:
+          [1701251118.633663][8779:8781] CHIP:DMG:         ],
+          [1701251118.633669][8779:8781] CHIP:DMG:
+          [1701251118.633672][8779:8781] CHIP:DMG:         InteractionModelRevision = 11
+          [1701251118.633675][8779:8781] CHIP:DMG: }
+          [1701251118.633729][8779:8781] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0062 Attribute 0x0000_0002 DataVersion: 3791900006
+          [1701251118.633743][8779:8781] CHIP:TOO:   FabricSceneInfo: 1 entries
+          [1701251118.633758][8779:8781] CHIP:TOO:     [1]: {
+          [1701251118.633765][8779:8781] CHIP:TOO:       SceneCount: 0
+          [1701251118.633768][8779:8781] CHIP:TOO:       CurrentScene: 0
+          [1701251118.633770][8779:8781] CHIP:TOO:       CurrentGroup: 0
+          [1701251118.633773][8779:8781] CHIP:TOO:       SceneValid: FALSE
+          [1701251118.633776][8779:8781] CHIP:TOO:       RemainingCapacity: 7
+          [1701251118.633779][8779:8781] CHIP:TOO:       FabricIndex: 1
+          [1701251118.633783][8779:8781] CHIP:TOO:      }
       disabled: true
 
     - label:
           "Step 2c:  Keep subscription session active for the remaining of the
           test"
       verification: |
+          Keep subscription session active for the remaining of the test
+
           Confirm that the subscription session is active for the remainder of the test by the reception of Liveness checks:
 
           CHIP:EM: >>> [E:44665r S:45698 M:201545516] (S) Msg RX from 1:0000000000000001 [1042] --- Type 0001:05 (IM:ReportData)
           CHIP:EM: Handling via exchange: 44665r, Delegate: 0xaaaaea044808
           CHIP:DMG: ReportDataMessage =
           CHIP:DMG: {
-          CHIP:DMG: 	SubscriptionId = 0xce6a96bc,
-          CHIP:DMG: 	InteractionModelRevision = 11
+          CHIP:DMG:         SubscriptionId = 0xce6a96bc,
+          CHIP:DMG:         InteractionModelRevision = 11
           CHIP:DMG: }
           CHIP:DMG: Refresh LivenessCheckTime for 9224 milliseconds with SubscriptionId = 0xce6a96bc Peer = 01:0000000000000001
       disabled: true
 
     - label: "Step 2d:  Repeat Step 2b and 2c with TH2 and TH3"
       verification: |
           TH2:
-          CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0005 Attribute 0x0000_0007 DataVersion: 2793536968
-          CHIP:TOO:   FabricSceneInfo: 1 entries
-          CHIP:TOO:     [1]: {
-          CHIP:TOO:       SceneCount: 0
-          CHIP:TOO:       CurrentScene: 0
-          CHIP:TOO:       CurrentGroup: 0
-          CHIP:TOO:       SceneValid: FALSE
-          CHIP:TOO:       RemainingCapacity: 7
-          CHIP:TOO:       FabricIndex: 2
-          CHIP:TOO:      }
+
```

Comment:
Follow up (please create an issue for tracking purposes) this step needs to separated for TH2 and TH3 in test spec. Overloading is error prone for manual tests